kctf-operator/deploy/crds/kctf.dev_v1alpha1_challenge_cr_example_1.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kctf.dev/v1alpha1
+kind: Challenge
+metadata:
+  name: mychal
```

Comment:
So, actually I was checking about the namespaces today. If we declare here the namespace, it will be the one used for the deployment and etc. Also, the operator can watch all namespaces. But I don't think it's ideal to declare the namespace there, right? And, just to confirm, the operator should use as default the name of the challenge as namespace, right?

The only thing that may be a problem now is that the namespace that we put there must already exist (at least with this code). I think it's the operator's duty to create a namespace for each challenge, right? I will do that next, but still need to figure out how to create one using Go.